README.md

```diff
@@ -336,36 +341,36 @@ You can pass any of `devnet`, `alphanet`, `betanet`, `testnet` or `mainnet` for
 There are plenty of options available that you can use to override configuration on runtime while starting the lisk.
 
 ```
-node app.js [options]
+npm start -- [options]
 ```
 
 Each of that option can be appended on command line. There are also few `ENV` variables that can be utilized for this purpose.
 
-| Option                               | ENV Variable           | Config Option            | Description                                                                                                                                                                       |
```

Comment:
Here I think your formatting changed.

Reply:
I previewed this with markdown plugin, looks the same as before.